### ISSUE TITLE
Semi-Supervised Deep Transfer Learning for Benign-Malignant Diagnosis of Pulmonary Nodules in Chest CT Images

### ISSUE THREAD

**Issue by ghuijo** [One-Day-One-Paper/One-Day-One-Paper#185]
*Semi-supervised Deep Transfer Learning (SDTL) framework for benign-malignant pulmonary nodule diagnosis*
##
### METHODS
* SDTL 모델 프레임워크
  <image src="https://user-images.githubusercontent.com/67627471/169700041-9dbf67a9-3eea-45f1-bb03-0ece84f022b0.png" width="70%" height="70%"/>
* Step 1: Image Preprocessing and Nodule Extraction
  * all CT images were converted into Hounsfield units
  * resampled into 1.0 mm slice thickness using trilinear interpolation
  * The intensity of each voxel (IH u ) was first normalized to I by Z-score standardization method
    and then clipped to an intensity range of [−1, 1]
  * to enhance the diversity of training data and strengthen the generalizability of the model, image augmentation methods were also adopted, including random flipping, rotation, and scaling
* Step 2: Transfer Learning by Using a Pre-Trained Network
  * 적은 양의 샘플로 인한 오버피팅을 방지하기 위해 transfer learning 전략 적용함
  * CNN model contains one input block, four downsample blocks, and one output block
* Step 3: Semi-Supervised Learning for Utilizing Unlabeled Data
  * there is a large number of nodules available in CT images with no pathological results
  * --> To address the high expense of acquiring annotated data, semi-supervised learning provides a potential solution to alleviate the insufficient annotation problem
  * the model was initially trained with a transfer learning strategy on the dataset with pathology labels.
  * the model was used to estimate labels for unlabeled nodules, which were then selectively added to the training dataset for refining the model

### RESULTS
* training 전략 별 classification model 성능 비교
  <image src="https://user-images.githubusercontent.com/67627471/169700116-8bfcf524-73c6-4d44-ad78-a05e7fdb35ab.png" width="80%" height="80%"/>
* semi-supervised learning 방법 별 분류 성능 비교
  <image src="https://user-images.githubusercontent.com/67627471/169700192-ad5c20b0-9043-4d34-b6ff-52266fccf936.png" width="60%" height="60%"/>
